### PR TITLE
Create Data Import

### PR DESCRIPTION
**Pull request by jenlu33**
Asana ticket: [https://app.asana.com/0/1203289004376659/1203289004376671/f](https://app.asana.com/0/1203289004376659/1203289004376671/f)

### CODE REVIEW

File: app/models/data_import.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+  validate :file_presence
+
+  private
+
+  def file_presence
+    unless file.attached?
+      errors.add(:file, "Must upload file")
+    end
+  end
```
Comment (littleforest) [inline app/models/data_import.rb:14]: I tested this in the UI, and it doesn't seem that our file validation is working.

File: app/views/data_imports/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      <div class="show-label">File</div>
+        <div class="w-full mb-1">
+          <%= link_to @data_import.file.filename.to_s, url_for(@data_import.file), class: "text-blue-600 hover:text-blue-800 visited:text-purple-600" %>
+        </div>
+    </div>
+  </div>
+  <div class=" max-w-prose mx-auto flex text-center space-x-2">
+    <%= link_to "Upload more standards", new_data_import_path, class: "btn-primary" %>
+    <%= link_to "Edit occupation standard", edit_data_import_path(@data_import), class: "btn-primary" %>
+    <%= button_to "Delete occupation standard", data_import_path(@data_import), method: :delete, data: {turbo_method: :delete}, class: "btn-primary" %>
```
Comment (littleforest) [inline app/views/data_imports/show.html.erb:18]: Ah nice, you got this to work!

**Review by littleforest — COMMENTED** [review]
@jenlu33 this is looking great! Could we round it out with adding the index method and adding a link in the navbar?

File: app/models/data_import.rb (hunk 1 above)
Comment (jenlu33) [reply app/models/data_import.rb:14]: @littleforest I added the error message into the views and it looks like the data import is not being created unless a file is attached

**Pull request merged**